projectmanagement/src/main/java/life/qbic/projectmanagement/application/ProjectInformationService.java

```diff
@@ -66,6 +67,10 @@ public void updateTitle(String projectId, String newTitle) {
     });
   }
 
+  public void manageProject(String projectId, PersonReference personReference) {
+    //TODO
```

Comment:
todo?

Reply:
Yes this is the issue I was talking about. This still needs to be done :D